docs/operator-manual/cluster-bootstrapping.md

```diff
@@ -83,7 +159,9 @@ argocd app create apps \
 argocd app sync apps  
 ```
 
-The parent app will appear as in-sync but the child apps will be out of sync:
+After syncing, the parent application will appear as Synced. However, the child applications will initially appear as `OutOfSync`, since they have not yet been created in the cluster, hence they must be reconciled individually.
```

Comment:
```suggestion
After syncing, the parent Application will appear as `Synced`. However, the child applications will initially appear as `OutOfSync`, since they have not yet been created in the cluster, hence they must be reconciled individually.
```